test/e2e/angular/font-setting-test-ng.js

```diff
@@ -275,14 +275,14 @@
       });
 
       describe("Google Font", function() {
-        it("should update font family for preview text if family name is one word", function() {
+        xit("should update font family for preview text if family name is one word", function() {
```

Comment:
@santiagonoguez Curious, why did these two tests need turning off?

Reply:
@stulees The problem was that the selector id used in the tests no longer exist in the component; 

I changed the selection by font name instead so it depends less on ids that we do not control.

Also, I changed the fonts used for these tests as previous one require scrolling on the select box before clicking. If we want to use the original fonts for these tests, that behavior should be coded into them also, but that may be another PR.

Reply:
@santiagonoguez The problem with this I see now is that these tests are specifically for Google fonts, and now you're targeting Arial. 

In the past, if we needed to find the selector id (due to a Google font order change), we'd test the component manually in browser and inspect the HTML to find the appropriate selector ID. 

I'd prefer we do this but don't want you to spend more time with this, so perhaps we go back to turning the tests off and revisit. Cheers

Reply:
@stulees I added code to scroll to the bottom of the font layer, and select Google fonts there. Please review again.